src/ucp/proto/proto_debug.c

```diff
@@ -219,7 +219,8 @@ ucp_proto_select_elem_info(ucp_worker_h worker,
     ucp_proto_select_param_dump(worker, ep_cfg_index, rkey_cfg_index,
                                 select_param, ucp_operation_descs, &ep_cfg_strb,
                                 &sel_param_strb);
-    if (!ucp_proto_debug_is_info_enabled(
+    if (!show_all &&
```

Comment:
should not it be `||`?

Reply:
It should be `&&`.
I think the name of the parameter may be misunderstood.
If `show_all` is true, we print information.
Otherwise, `show_all` is false, and if `PROTO_INFO=y`, we print the information.